### DISCUSSION TITLE
ABB Charger with connected backoffice software

### DISCUSSION THREAD

**Discussion by whizzkid1978** [lbbrhzn/ocpp#869] [General]
Hi all,

I have a supported ABB EV Charger. This one is already connected to an OCPP server of my backoffice software (e-flux.io). Via this backoffice software it's possible to charge my employee.

But I also want to connect the EV charger with Home Assistant combined with an integration of my local energy company (Frank). Dynamic charging work with my stekker.app. I want to connect the charging point with Home Assistant to have insights in the costs per charge session, so that I am sure that I charge my employee the right energy prices.

But in my ABB charging point I can only connect with one OCPP server. I can't add a second OCPP server (like Home Assistant).

Do you know another way around to solve my question? :-)

Thanks in advance?

**Comment by RafSchandevyl** [comment]
Would it be possible to create some kind of proxy CS.  This would forward all local request to a other central station.
